test/server/cards/01_SOR/units/ISBAgent.spec.ts

```diff
@@ -0,0 +1,75 @@
+describe('ISB Agent', function() {
+    integration(function(contextRef) {
+        describe('ISB Agent\' ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['confiscate', 'waylay', 'isb-agent'],
+                        groundArena: ['atst'],
+                        spaceArena: ['cartel-spacer']
+                    },
+                    player2: {
+                        hand: ['disarm'],
+                        groundArena: ['wampa'],
+                        spaceArena: ['alliance-xwing']
+                    }
+                });
+            });
+
+            describe('when event is revealed', function() {
+                it('should deal 1 damage to a unit', function () {
+                    const { context } = contextRef;
+
+                    context.player1.clickCard(context.isbAgent);
+
+                    expect(context.player1).toBeAbleToSelectExactly([context.confiscate, context.waylay]);
+                    expect(context.player1).toHaveExactPromptButtons(['Choose no target']);
+                    context.player1.clickCard(context.confiscate);
+
+                    expect(context.getChatLogs(3)).toEqual([
+                        'player1 plays ISB Agent',
+                        'player1 uses ISB Agent to reveal a card',
+                        'player1 reveals Confiscate due to ISB Agent',
+                    ]);
+
+                    expect(context.player1).toBeAbleToSelectExactly([context.isbAgent, context.atst, context.cartelSpacer, context.wampa, context.allianceXwing]);
+                    context.player1.clickCard(context.wampa);
+
+                    expect(context.isbAgent).toBeInLocation('ground arena');
+                    expect(context.isbAgent.damage).toBe(0);
+                    expect(context.atst.damage).toBe(0);
+                    expect(context.cartelSpacer.damage).toBe(0);
```

Comment:
Never opposed to verbose test cases, but if you prefer writing fewer lines we generally don't bother checking that units except the target have damage. We have enough confirmation that the targeting + damage pipeline works across the test suite

Reply:
Yeah, makes sense! Removed :)